src/snowflake/cli/_plugins/notebook/commands.py

```diff
@@ -84,3 +103,54 @@ def create(
         notebook_file=notebook_file,
     )
     return MessageResult(message=notebook_url)
+
+
+@app.command(requires_connection=True)
+@with_project_definition()
+def deploy(
+    entity_id=entity_argument("notebook"),
+    replace: bool = ReplaceOption(
+        help="Replace existing Notebook if it already exists.",
+    ),
+    if_not_exists: bool = IfNotExistsOption(help="Skip if Notebook already exists."),
+    **options,
+) -> CommandResult:
+    """Uploads a notebook to a stage and creates a notebook. If entity_id is not provided,
+    deploys all notebooks defined in the project definition."""
```

Comment:
```suggestion
    deploys all notebooks defined in the project definition."""
```
See how streamlit works. Also consider borrowing this helper:
https://github.com/snowflakedb/snowflake-cli/blob/347c46a8ec7a186b405e67041959e90ffc0af097/src/snowflake/cli/_plugins/streamlit/commands.py#L158-L163

Reply:
Fixed, nice helper!